### ISSUE TITLE
Executing commands on a slave zone

### ISSUE THREAD

**Issue by DPH** [SoCo/SoCo#138]
Many commands only apply to a master (coordinator) zone and you would not expect a user to execute them on a slave. However people do! 
Below is a list of commands tested against a slave zone, and at the bottom one against a zone that has volume set to fixed (typically a ZP90 driving an external amp). It shows the command and the behaviour / error that occurs.

Maybe this behaviour is what we should expect, and users should trap it?
Or should we do nothing and just return (as play and stop do)?
Or should we raise a trappable exception with a clear message that it should not be executed on a slave?
Or......

Welcome thoughts
Cheers David

```
1) commands executed on a slave:

    play               does nothing - ok?
    stop               does nothing - ok?

    play_uri           un_joins zone to be a master and plays -ok?
    play_from_queue    un_joins zone to be a master and plays -ok?
    switch_to_line_in  un_joins zone to be a master and sets to line in -ok?

    pause              soco.exceptions.SoCoUPnPException: UPnP Error 701 received: Transition not available from 192.168.1.69
    seek               soco.exceptions.SoCoUPnPException: UPnP Error 701 received: Transition not available from 192.168.1.69
    next               soco.exceptions.SoCoUPnPException: UPnP Error 701 received: Transition not available from 192.168.1.69
    previous           soco.exceptions.SoCoUPnPException: UPnP Error 701 received: Transition not available from 192.168.1.69
    partymode          soco.exceptions.SoCoUPnPException: UPnP Error 501 received: Action Failed from 192.168.1.72
    add_to_queue       soco.exceptions.SoCoUPnPException: UPnP Error 800 received:  from 192.168.1.69
    remove_from_queue  soco.exceptions.SoCoUPnPException: UPnP Error 800 received:  from 192.168.1.69
    clear_queue        soco.exceptions.SoCoUPnPException: UPnP Error 800 received:  from 192.168.1.69
    
2) command executed on zones with fixed volume set: 
    volume             soco.exceptions.SoCoUPnPException: UPnP Error 501 received: Action Failed from 192.168.1.71
```

**Comment by KennethNielsen** [comment]
please check out #296 :)

**Comment by KennethNielsen** [comment]
This should be fixed with #296 as found the latest release 0.11. Please re-open if that is not the case.

**Comment by AdrienPensart** [comment]
This simply does not work : 

```
#!/usr/bin/env python
from soco import SoCo

if __name__ == '__main__':
    sonos = SoCo('10.65.0.167')                        
    sonos.clear_queue()
    sonos.play_uri('http://127.0.0.1:8000/12.Maps.mp3')
    sonos.play()
```

outputs : 

```
> $ python play.py                                                                                   
Traceback (most recent call last):
  File "play.py", line 17, in <module>
    sonos.play()
  File "build/bdist.linux-x86_64/egg/soco/core.py", line 439, in play
  File "build/bdist.linux-x86_64/egg/soco/services.py", line 158, in _dispatcher
  File "build/bdist.linux-x86_64/egg/soco/services.py", line 349, in send_command
  File "build/bdist.linux-x86_64/egg/soco/services.py", line 409, in handle_upnp_error
soco.exceptions.SoCoUPnPException: UPnP Error 701 received: Transition not available from 10.65.0.167
```

**Comment by KennethNielsen** [comment]
Hallo @AdrienPensart

And thanks for you message.

It does not seem to me that this is related to the discussion above, which deals explicitly with the problem of executing commands on a zone that is a slave.

Please open a separate issue for your problem.

Regards Kenneth
